src/de/False/BuildersWand/ConfigurationFiles/Locales.java

```diff
@@ -111,8 +111,14 @@ private void setDefaults()
         de_de.put("reload", "Die Konfig wurde neugeladen.");
         de_de.put("playerNotFound", "Der Spieler konnte nicht gefunden werden.");
         de_de.put("noPermissions", "Du besitzt zu wenige Rechte um dies zu tun.");
+        
+		Map<String, String> es_es = new HashMap<>();
```

Comment:
Pls remove the unnecessary whitespace in front of the map.

Reply:
sorry boss :(